api_v3/services/BaseEntryService.php

```diff
@@ -153,6 +153,14 @@ protected function attachResource(kResource $resource, entry $dbEntry, asset $as
 				$service = new LiveStreamService();
     			$service->initService('liveStream', 'liveStream', $this->actionName);
     			break;
+		    case entryType::DOCUMENT:
+				if(KalturaPluginManager::getPluginInstance('document'))
```

Comment:
if the plugin is not enabled and the entry is doc type need to throw the error like we used to and not continue to the logic afterward

Reply:
when the plugin is not enabled, the codition drops to the default case like before